tests/oclinter/key-quoting/openconfig-testcase-succeed.yang

```diff
@@ -29,7 +29,8 @@ module openconfig-testcase-succeed {
 
   grouping testcase-top {
     description
-      "Top-level grouping";
+      "Top-level grouping that uses the 
+      word key in it";
```

Comment:
do you mean grouping that uses a list with a key statement ?

Reply:
No, it's to catch the case that we have the word key that is not within a key statement. It's to check that we don't error in that case.